src/box/memtx_hash.c

```diff
@@ -433,10 +433,17 @@ memtx_hash_index_create_iterator(struct index *base, enum iterator_type type,
 			light_index_iterator_begin(&index->hash_table, &it->iterator);
 			it->base.next = hash_iterator_ge;
 		}
+		/* This iterator needs to be supported as a legacy. */
+		memtx_tx_track_full_scan(in_txn(),
```

Comment:
This function can fails, may be handle this?

Reply:
Actually I don't know what to do in case of that error. Iterators don't have errors now. We cannot just stop iteration - it would be quite unexpected.